modules/apps/commerce/BREAKING_CHANGES.markdown

```diff
@@ -397,4 +397,33 @@ CommercePricingClassScreenNavigationConstants;`
 
 This was moved to follow Liferay coding pattern.
 
+---------------------------------------
+
+### Destination Names Changed
+- **Date:** 2020-Sep-10
+- **JIRA Ticket:** [COMMERCE-4762](https://issues.liferay.com/browse/COMMERCE-4762)
+
+#### What changed?
+
+The prefix commerce_ has been added to the commerce destinations defined in
```

Comment:
We should make it a little clearer that commerce_ is the prefix, either by formatting it as `commerce_` or containing it within quotes, i.e. "commerce_"

We should follow whatever pattern is already established by the Portal breaking changes doc.